api/control.proto

```diff
@@ -88,6 +88,7 @@ message ListNodesRequest {
 		map<string, string> labels = 3;
 		repeated NodeSpec.Membership memberships = 4;
 		repeated NodeRole roles = 5;
+		repeated string name_prefixes = 6 [(gogoproto.customname) = "NamePrefixes"];
```

Comment:
Let's make it

    // NamePrefixes matches all objects with the given prefixes

slightly more to the point